uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/carddetailscheckout/CardDetailsCheckoutScreen.kt

```diff
@@ -594,17 +622,23 @@ private fun HeaderItem(state: CardDetailsCheckoutState) {
 }
 
 @Composable
-private fun AppBarItem(onBackClicked: () -> Unit, onCloseClicked: () -> Unit, toolbarTitle: String?) {
+private fun AppBarItem(
+    onBackClicked: () -> Unit,
+    onCloseClicked: () -> Unit,
+    toolbarTitle: String?,
+) {
     DojoAppBar(
-        title = toolbarTitle ?: stringResource(id = R.string.dojo_ui_sdk_card_details_checkout_title),
+        title = toolbarTitle
+            ?: stringResource(id = R.string.dojo_ui_sdk_card_details_checkout_title),
         titleGravity = TitleGravity.LEFT,
         navigationIcon = AppBarIcon.back(DojoTheme.colors.headerButtonTintColor) { onBackClicked() },
         actionIcon = AppBarIcon.close(DojoTheme.colors.headerButtonTintColor) { onCloseClicked() },
     )
 }
 
 private const val FIRST_FIELD_OFF_SET_DP = 30
-private const val SECOND_FIELD_OFF_SET_DP = 290
-private const val THIRD_FIELD_OFF_SET_DP = 360
-private const val FORTH_FIELD_OFF_SET_DP = 420
+private const val SECOND_FIELD_OFF_SET_DP = 230
+private const val THIRD_FIELD_OFF_SET_DP = 330
+private const val FORTH_FIELD_OFF_SET_DP = 430
 private const val FIFTH_FIELD_OFF_SET_DP = 490
+private const val SIXTH_FIELD_OFF_SET_DP = 550
```

Comment:
we have to make this dynamic, it's not good at all having values fixed like this. If you change device resolutions or font size this might break it.

Reply:
for that one, those values are in DP not Px, for sure I need to make that one dynamic but I will add those of future PRs as it's part of the refactor plan that we have